src/data.js

```diff
@@ -263,45 +263,154 @@ let Artifact = Entity.configure({
     name:           Entity.types.String,
     storageType:    Entity.types.String,
     contentType:    Entity.types.String,
+    details:        Entity.types.JSON,
+    expires:        Entity.types.Date,
+  },
+  context: [
+    'blobStore',
+    'privateBucket',
+    'publicBucket',
+    'monitor',
+  ],
+});
+
+/** Entity for tracking artifacts.
+ * Version 2 adds 'present' field */
+let Artifact = Entity.configure({
```

Comment:
You can do `Artifact = Artifact.configure({version: 2`, but just use the format we had before...

Also you can't do `let Artifact` twice :)